src/functions/calcTotalPoints.jsx

```diff
@@ -6,12 +6,13 @@ function CalcPoints () {
   // console.log("calc points called");
   // console.log(JSON.stringify(allUserScores));
   const userScores = useMemo(() => {
+    if (!(Array.isArray(allUserScores) && currentUser)) return [];
+
     return allUserScores.filter(scores => {
         return scores.user && scores.user._id === currentUser._id
             && scores.round === round
     });
-
-  }, [allUserScores, currentUser._id, round]);
```

Comment:
`currentUser` may be `undefined` depending on whether the user is logged in or not, or if there is user session stored in `sessionStorage`. Therefore, trying to read `currentUser._id` here would cause console errors sometimes and could even lead to compile & render issues.

I changed it to `currentUser` instead